src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -782,8 +788,6 @@ GrpcLb::BalancerCallState::BalancerCallState(
                     OnBalancerMessageReceived, this, grpc_schedule_on_exec_ctx);
   GRPC_CLOSURE_INIT(&lb_on_balancer_status_received_, OnBalancerStatusReceived,
                     this, grpc_schedule_on_exec_ctx);
-  GRPC_CLOSURE_INIT(&client_load_report_closure_, MaybeSendClientLoadReport,
```

Comment:
Might as well initialize this closure here to point to `ClientLoadReportDone` instead of doing so in `SendClientLoadReportLocked()`, since it now only needs to be set once, not once for each load report.